go/store/nbs/store_test.go

```diff
@@ -334,14 +334,18 @@ func TestNBSCopyGC(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, ok)
 
-	keepChan := make(chan []hash.Hash, numChunks)
+	require.NoError(t, st.BeginGC(nil))
```

Comment:
is our GC testing this sparse? or do we have tests at other interface levels somewhere else